x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/components/auto_follow_pattern_table/auto_follow_pattern_table.js

```diff
@@ -89,29 +90,21 @@ export class AutoFollowPatternTableUI extends PureComponent {
       ),
       render: (leaderPatterns) => leaderPatterns.join(', '),
     }, {
-      field: 'followIndexPattern',
+      field: 'followIndexPatternPrefix',
```

Comment:
Can we add `sortable: true` to the prefix and suffix columns?

Reply:
Yes, don't know why I thought it was already set to `true` 😊